tests/test_app.py

```diff
@@ -445,13 +445,16 @@ def test_simple_index_list_name_with_underscore_no_egg(root, testapp):
 def test_json_info(root, testapp):
     root.join("foobar-1.0.zip").write("")
     root.join("foobar-1.1.zip").write("")
+    root.join("foobar-1.1-linux.zip").write("")
     root.join("foobarX-1.1.zip").write("")
 
     resp = testapp.get("/foobar/json")
     assert "info" in resp.json
     assert "releases" in resp.json
     assert len(resp.json["info"]) == 1
     assert len(resp.json["releases"]) == 2
+    assert len(resp.json["releases"]["1.0"]) == 1
+    assert len(resp.json["releases"]["1.1"]) == 2 
```

Comment:
Oh oups, `black` has spotted a small formatting thing: 

```suggestion
    assert len(resp.json["releases"]["1.1"]) == 2
```

After that, it's good to go! ✌️